JitsiConferenceEventManager.js

```diff
@@ -739,6 +739,11 @@ JitsiConferenceEventManager.prototype.setupStatisticsListeners = function() {
             JitsiConferenceEvents.BEFORE_STATISTICS_DISPOSED);
     });
 
+    conference.statistics.addEncodeTimeStatsListener((tpc, stats) => {
```

Comment:
Commenting here, semi-randomly: looks like this commit could be extracted from this PR to make it smaller, it can be applied to the current code, can't it?

Reply:
It can be applied to the current code but the client can act on them only if the codec selection API is used. It didn't make sense to capture/process the encodeTime stats if we are not acting on them.